app/src/main/java/com/intive/selftraining/selftraining/listmovies/adapter/ItemsAdapter.kt

```diff
@@ -2,7 +2,6 @@
 
 package com.ciandt.recyclerviewbinding.presentation.items
```

Comment:
we need to remove 
@file:JvmName("ItemsAdapter")
and add correct package instead of
package com.ciandt.recyclerviewbinding.presentation.items